tests/sles4sap/qesapdeployment/configure.pm

```diff
@@ -30,6 +30,7 @@ sub run {
     $variables{SSH_KEY_PRIV} = '/root/.ssh/id_rsa';
     $variables{SSH_KEY_PUB} = '/root/.ssh/id_rsa.pub';
     $variables{SCC_REGCODE_SLES4SAP} = get_required_var('SCC_REGCODE_SLES4SAP');
+    $variables{HANA_INSTANCE_TYPE} = get_var('QESAP_HANA_INSTANCE_TYPE', 'r6i.xlarge');
```

Comment:
The default value match this one https://github.com/SUSE/qe-sap-deployment/blob/accc817d3cd1d606afa65c136f18b432bff2797f/terraform/aws/variables.tf#L191

Reply:
Should the default value 'default = "r6i.xlarge"' be changed according to https://jira.suse.com/browse/TEAM-7291?

Reply:
Probably the values mentioned in the ticket are to be contained in `QESAP_HANA_INSTANCE_TYPE` var, and default to be used only should this var is not specified

Reply:
yes, this PR is just the first step to be able to create a JobGroup testing all the types